Unix/mak/rules.mak

```diff
@@ -275,12 +275,13 @@ TARGET = $(LIBDIR)/$(shell $(BUILDTOOL) shlibname $(CSHLIBRARY))
 
 __LIBRARIES = $(addprefix -l,$(LIBRARIES))
 __LIBRARIES += $(addprefix -l,$(SYSLIBRARIES))
+__DEPS=$(wildcard $(addprefix $(LIBDIR)/lib,$(addsuffix .*,$(LIBRARIES))))
```

Comment:
This is consistent for both `CSHLIBRARY` and `CXXSHLIBRARY`. Rather than copy/paste, you might want to consider just defining this rule once.